automation/scripts/generate-keys-and-ledger.sh

```diff
@@ -360,7 +374,7 @@ add_another_to_prompt ${TESTNET}_offline-fish ${FISH_AMOUNT} ${TESTNET}_online-f
 add_another_to_prompt ${TESTNET}_online-fish ${FISH_AMOUNT} ${TESTNET}_online-fish
 add_another_to_prompt ${TESTNET}_online-o1 ${FISH_AMOUNT} ${TESTNET}_online-o1
 
-if [ -d keys/keysets/${TESTNET}_bots_keyfiles ];
+if [ -f keys/keysets/${TESTNET}_bots_keyfiles ];
```

Comment:
I think this should still be `-d` as ${TESTNET}_bots_keyfiles should be a directory https://wiki.bash-hackers.org/commands/classictest#file_tests

Reply:
on my local system `${TESTNET}_bots_keyfiles` is a file tho